### DISCUSSION TITLE
v1.22.1

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#619] [Release]
**Features:**
Always print masterserver response request id ([Launcher#633](https://github.com/R2Northstar/NorthstarLauncher/pull/633))
Add slow-speed timeout when downloading mod ([Launcher#611](https://github.com/R2Northstar/NorthstarLauncher/pull/611))

**Fixes:**
Fix debug overlay not rendering script function stuff ([Launcher#637](https://github.com/R2Northstar/NorthstarLauncher/pull/637))
Escape backticks in run command ([#617](https://github.com/R2Northstar/Northstar/pull/617))
Prevent client side crash in MFD when marked target dies out of bounds. ([Mods#705](https://github.com/R2Northstar/NorthstarMods/pull/705))

**Changes:**
Remove `structs.h` in favor of manual padding ([Launcher#613](https://github.com/R2Northstar/NorthstarLauncher/pull/613))
Add and use ScopeGuard ([Launcher#643](https://github.com/R2Northstar/NorthstarLauncher/pull/643))

**Contributors:**
@Alystrasz @ASpoonPlaysGames @GeckoEidechse @Jan200101

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.22.1'>v1.22.1</a>.</em>

**Comment by Leo-byte502** [comment]
hello I have a problem with the launcher it does not let me create a network, there is a solution to this problem I would appreciate it very much

**Reply by GeckoEidechse** [reply]
Northstar doesn't support networks yet. If you wanna create a network etc you need to use vanilla still.

**Comment by Astarothcx8** [comment]


**Reply by uniboi** [reply]
Do you mean the Mods Menu in the Main Menu?
